Clicker/ViewControllers/PollsDateViewController.swift

```diff
@@ -128,20 +128,13 @@ class PollsDateViewController: UIViewController {
     
     @objc func goBack() {
         socket.socket.disconnect()
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+        self.navigationController?.popViewController(animated: true)
```

Comment:
do we want this to always happen? I think that if you pop the view controller before deleting, the home screen will still show the deleted poll.

Reply:
I did this because if you need to delete the poll then there is a noticeable wait for the user before they see the app go back. Although, you are right that this way the home screen will show the deleted poll.